types/animation.d.ts

```diff
@@ -0,0 +1,31 @@
+import { Chart } from './index.esm';
```

Comment:
If I don't import this here, when combined the definition of `Chart` will be `Chart$1`